requirements/edx/base.txt

```diff
@@ -189,6 +189,7 @@ py2neo==3.1.2
 pycontracts==1.7.1
 pycountry==18.12.8
 pycparser==2.19
+pycryptodome==3.8.0       # via pdfminer.six
 pycryptodomex==3.4.7
```

Comment:
Sigh...we now have effectively the same package installed under two different directory names, since the `pyjwkest` package chose to use `pycryptodomex` instead.